EYBlockchain/nightfall_3
# David/wallet error msg

Show error message in wallet when wallet expects to connect to localhost and metamask is connected to a different network, It also shows error message when a transaction fails.

This PR also loads erc contract addresses from contract name and not from an expected address. 

NOTE: You must clear local storage

To test it:
- connect metamask to Ropsten network, start the wallet, and try to log in. An error message should appear
- Send a deposit without having reset the metamask nonce => An error message should appear
- Send a transfer/withdraw without any valid commitment => An error message should appear

Fixes #221
